sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Window.java

```diff
@@ -486,7 +486,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
           // We first apply a (trivial) transform to the input PCollection to produce a new
           // PCollection. This ensures that we don't modify the windowing strategy of the input
           // which may be used elsewhere.
-          .apply("Identity", MapElements.via(new SimpleFunction<T, T>() {
+          .apply("Identity", MapElements.<T,T>via(new SimpleFunction<T, T>() {
```

Comment:
Thanks for your patience, you're right, my memory was wrong about error message.

Here's the error:
![screenshot from 2017-04-25 19-47-42](https://cloud.githubusercontent.com/assets/35872/25419754/32914bde-29f0-11e7-95bd-2f5b97afb7c3.png)

IDE is actually Android Studio, which I thought was the same as IntelliJ because they mostly behave identically, but I am realizing may not be given that it doesn't have the same settings for "annotation processing" as documented here: https://beam.apache.org/contribute/contribution-guide/#intellij 

Does Beam only intentionally support language level 7?  I noticed when I changed this setting from 8=>7 that the error went away:

![screenshot from 2017-04-25 20-02-14](https://cloud.githubusercontent.com/assets/35872/25420160/64387fb6-29f2-11e7-85d9-88abf613c028.png)

Totally happy to have this PR get closed if I'm doing something wrong!